vmngclient/tests/test_templates.py

```diff
@@ -61,238 +63,175 @@ def setUp(self):
         self.task = TaskStatus("Success", "success", [])
 
     @patch("vmngclient.session.vManageSession")
-    def test_templates_success(self, mock_session):
+    @patch("vmngclient.response.vManageResponse")
```

Comment:
Order of the @patch'es should be like in decorators - in this case I believe that `mock_session` argument in test is mocking `vmngclient.response.vManageResponse`